chapters/en/slides/module3/module3_02.Rmd

```diff
@@ -0,0 +1,308 @@
+---
+params:
+  dynamictitle: "module3_02"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+# Using density plots to visualize distributions
+
+Notes:
+In this module we will see how to use density plots
+instead of histograms to represent distributions of data.
+
+---
+
+## Histograms can give different results depending on the data
+
+<img src=/module3/histogram-and-shifted.png></img>
+
+[From scikit-learn](https://scikit-learn.org/stable/auto_examples/neighbors/plot_kde_1d.html)
+
+Notes:
+How many observations are counted per bar in a histogram
+depends on exactly where on the axis the border between the bins are.
+
+In the images on this slide,
+the actual observations are drawn with black tick marks on the bottom
+and they are the same in both subplots.
+
+The reason the histograms look different
+is that the border between the groups is shifted in the rightmost picture.
+
+As we see here,
+a histogram is not an as unbiased plot as we might think at first,
+especially not if we have few data points,
+where the inclusion or exclusion of just a few points
+makes a big difference for the bar height.
+
+---
+
+## Centering the bins on the data can help create more accurate distribution plots
+
+<img src=/module3/kde-buildup.gif></img>
+
+Notes:
+Instead of setting fixed lines along the axis
+and then count plots fully in one bin or another,
+we can create bins that are centered
+on the data and then add them together.
+
+When we center bins on the data,
+we also usually have a round shape 
```

Comment:
I feel like more needs to be explained here.. Why do we give them a round shape? And it's not really round, its bell-shaped...

Reply:
I meant to say "rounded" I think. So this part I have simplified based on my understanding of the learners capabilities and previous comments on some section being too difficult, but I am happy to include more detail. In 531 I do mention that there are different kernels and briefly show them but don't have any real discussion around it because "everyone" just uses gaussian kernal for KDEs without any real good reason, it just happens to be the default option. And while gaussian reduces noise compares to a square top had kernel, which makes it looks more aesthically pleasing, I don't know if it is technically "the best" choice". From what I have read, the kernel that best represents the distribution (whatever that really means), is often epanechnikov, but in my testing there is not a large difference vs gaussian so might as well stick with the default. [Some examples here](https://scikit-learn.org/stable/auto_examples/neighbors/plot_kde_1d.html#sphx-glr-auto-examples-neighbors-plot-kde-1d-py
)
 
What do you think would be a suitable level of detail to include here? Since they haven't heard of p-values and error bars, I imagine they don't know of gaussian/normal and mentioning any of these kernels by name is not useful. Something like the suggestion below maybe?

Reply:
I rephrased this section